module/modmanager/manager.go

```diff
@@ -260,7 +260,7 @@ func (m *module) checkReady(ctx context.Context, parentAddr string) error {
 }
 
 func (m *module) startProcess(ctx context.Context, parentAddr string, logger golog.Logger) error {
-	m.addr = filepath.Dir(parentAddr) + "/" + m.name + ".sock"
+	m.addr = filepath.ToSlash(filepath.Join(filepath.Dir(parentAddr), m.name+".sock"))
```

Comment:
I should've done that in the first place... I literally had the thought, then "no, we don't support windows anyway, so a simple slash literal is more readable"